pgx-pg-sys/src/submodules/errcodes.rs

```diff
@@ -0,0 +1,1291 @@
+use std::fmt::{Display, Formatter};
+
+/// This list of SQL Error Codes is taken directly from Postgres 12's generated "utils/errcodes.h"
```

Comment:
At some point we should find a way to generate this. Will think about it when I do some bindgen work.